packages/react-flicking/src/react-flicking/Flicking.tsx

```diff
@@ -0,0 +1,117 @@
+import NativeFlicking, { FlickingOptions, Plugin, FlickingEvent, NeedPanelEvent, withFlickingMethods, DEFAULT_OPTIONS } from "@egjs/flicking";
+import * as React from "react";
+import { findDOMNode } from "react-dom";
+import ChildrenDiffer from "@egjs/react-children-differ";
+import { CloneComponent } from "./Clone";
+import { FLICKING_PROPS } from "./consts";
+import { FlickingProps, FlickingType } from "./types";
+import ListDiffer from "@egjs/list-differ";
+import { ChildrenDiffResult } from "@egjs/children-differ";
+
+@withFlickingMethods
+class Flicking extends React.Component<Partial<FlickingProps & FlickingOptions>> {
+  public static defaultProps: FlickingProps = FLICKING_PROPS;
+  public state: {
+    cloneCount: number,
+  } = {
+      cloneCount: 0,
+    };
+  // Flicking
+  private flicking?: NativeFlicking | null;
+  private options: FlickingOptions = {
+    ...DEFAULT_OPTIONS,
+  };
+  // differ
+  private pluginsDiffer: ListDiffer<Plugin> = new ListDiffer<Plugin>();
+  // life cycle
+  public render() {
+    const props = this.props;
+    // tslint:disable-next-line:naming-convention
+    const Tag = props.tag as any;
+    const classPrefix = props.classPrefix;
+    const options = this.options;
+    const attributes: { [key: string]: any } = {};
+
+    for (const name in props) {
+      if (name in DEFAULT_OPTIONS) {
+        options[name] = props[name];
```

Comment:
@daybrush Are **all** options applied by this?

Reply:
`DEFAULT_OPTIONS` has type of `FlickingOptions`, so basically it'll cover all Flicking's options.

Reply:
@daybrush , @WoodNeck Sorry, I've provided too insufficient information. I mean... This option setting is occurred on every render loop. I wonder all option are activated whenever user has changed option dynamically. If not, It would be better to set on init time.

Reply:
That's right.
As native Flicking won't be affected by dynamic change of options object, option setting should be done only at initialization.
I remember @daybrush said he will update his code about it.